src/plugins/plugin.legend.js

```diff
@@ -106,6 +107,11 @@ var Legend = Element.extend({
 		// Contains hit boxes for each dataset (in dataset order)
 		this.legendHitBoxes = [];
 
+		/**
+ 		* @private
```

Comment:
this line and the next should be indented a bit more so that the first asterisk on each line of the comment align. you can look at other comments in the codebase for an example